### DISCUSSION TITLE
kanban & calendar view [Apr 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1677] [Discord Threads]
> 🙋‍♂️ **sangle**  
## I am running my nocodb with Heroku-1-click Hobby Dyno (just reinstalled today)  and couldn't find the options for kanban and calendar view. Please advise!  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/960684555357028363/Screen_Shot_2022-04-05_at_05.59.49.png)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### these views r coming-soon features. not available yet.  
#  
> 🙋‍♂️ **sangle**  
#### thank you wingkwong@discord got it 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/960684556032303134)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
